extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -294,7 +336,10 @@ public void accept(SecurityIdentity identity) {
                                                 finalRedirectUri += ("?" + finalUserQuery);
                                             }
                                             LOG.debugf("Final redirect URI: %s", finalRedirectUri);
-                                            uniEmitter.fail(new AuthenticationRedirectException(finalRedirectUri));
+                                            //TODO: Does this need to check autoRedirect
+                                            uniEmitter.fail(new AuthenticationRedirectException(
+                                                    shouldAutoRedirect(configContext, context),
+                                                    finalRedirectUri));
```

Comment:
Same here. This constructor doesn't exist for AuthenticationRedirectException. The original code at 297 is the correct version.